### PR TITLE
Task to load sip2 configs using a list of service point codes

### PR DESCRIPTION
**Pull request by jgreben**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ AllCops:
     - 'Gemfile'
     - 'Rakefile'
     - 'spec/spec_helper.rb'
+    - 'spec/circulation/load_sip2_settings_task_spec.rb'
```
Comment (jgreben) [inline .rubocop.yml]: Rubocop hates the json mock in this test in three different ways, but it has to be formatted exactly like it is in order to pass the test, so just exclude the entire test file from rubocop's gaze...

**Review by shelleydoljack — CHANGES_REQUESTED** [review]
Can the edge-sip2 configs be added to this task or use the configurations helper? https://github.com/sul-dlss/folio-tasks/blob/main/tasks/configurations/load_config_entries.rake

**Review by shelleydoljack — COMMENTED** [review]
I thought maybe you would just need to add 'edge-sip2' to the settings file for `configurations`, such as https://github.com/sul-dlss/folio-tasks/blob/26f187b6b6c025425c1f42ceb6b48fb25504e051/config/settings/test.yml#L19-L20 and it could be included in the `configurations:pull_configs` rake task after changing how filenames are created. Maybe for a refactoring PR. Do we need the acsTenantConfig too `/configurations/entries?query=module==edge-sip2 and configName==acsTenantConfig`?

**Comment by jgreben** [comment]
Given that I had to load it this way initially and that it looks up the ids based on the service points listed, I don't think we necessarily need to change all of it to pull from the "orig" platform.

**Pull request merged**